scripts/vits/inflect/nano_v2/export_onnx.py

```
@@ -0,0 +1,173 @@
+        "has_espeak": "1",
+        "version": 2,
+    }
+    print(meta_data)
+    add_meta_data(filename=str(model_path), meta_data=meta_data)
+
+
+def main() -> None:
+    torch.manual_seed(42)
+
+    if len(sys.argv) != 2:
+        raise RuntimeError(f"Usage: python3 ./export_onnx.py ./path/to/checkpoint_dir")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)